hangman/game.py

```diff
@@ -1,25 +1,79 @@
 from .exceptions import *
-
+import random
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
 
 def _get_random_word(list_of_words):
-    pass
+    #1. pick an integer from the len(list_of_words)
+    #2. return word in the list at the position of the integer
+    number_of_words = len(list_of_words)
+    if number_of_words == 0:
+        raise InvalidListOfWordsException
+    random_int = random.randint(0,number_of_words)
+    return list_of_words[random_int-1]
```

Comment:
you may use return random.choice(list_of_words) .. the choice() method will randomly pick from your list :)

can check out here for documentation - https://docs.python.org/3/library/random.html